conf/defaults.yaml

```diff
@@ -281,3 +281,16 @@ pacemaker.thread.timeout: 10
 pacemaker.childopts: "-Xmx1024m"
 pacemaker.auth.method: "NONE"
 pacemaker.kerberos.users: []
+
+storm.resource.isolation.plugin: "org.apache.storm.container.cgroup.CgroupManager"
+
+# Configs for CGroup support
+storm.cgroup.hierarchy.dir: "/cgroup/storm_resources"
+storm.cgroup.resources:
+    - cpu
+    - memory
+storm.cgroup.hierarchy.name: "storm"
+# Also determines whether the unit tests for cgroup runs.  If cgroup.enable is set to false the unit tests for cgroups will not run
```

Comment:
I think it would be better if this comment were part of the javadoc for the config definition in Config.java.  If we remove `storm.cgroup.enable` as @knusbaum [suggests](https://github.com/apache/storm/pull/1053/files#r52045815), then this comment should move to the plugin config too.